packages/tests/e2e/robot/tests/api/API_deploy_route_model.robot

```diff
@@ -5,42 +5,55 @@ ${RES_DIR}                          ${CURDIR}/resources/deploy_route_model
 ${PACKAGING}                        simple-model
 ${DEPLOYMENT}                       wine-api-testing
 ${MODEL}                            ${DEPLOYMENT}
-${MODEL_URL}                        ${EDGE_URL}/model/${MODEL}
 ${REQUEST}                          SEPARATOR=
 ...                                 { "columns": [ "a", "b" ], "data": [ [ 1.0, 2.0 ] ] }
 ${REQUEST_RESPONSE}                 { "prediction": [ [ 42 ] ], "columns": [ "result" ] }
 ${WrongHttpStatusCode}              SEPARATOR=
 ...                                 WrongHttpStatusCode: Got error from server: entity "{entity name}" is not found (status: 404)
+${WrongStatusCodeReturned}          SEPARATOR=
+...                                 Wrong status code returned: 404. Data: . URL: {model url}
+
+${DEPLOYMENT_NOT_EXIST}             deployment-api-not-exist
+${MODEL_NOT_EXIST}                  ${DEPLOYMENT_NOT_EXIST}
 
 *** Settings ***
-Documentation       API of training, packaging, deployment, route and model
+Documentation       API of deployment, route and model
 Resource            ../../resources/keywords.robot
 Resource            ./resources/keywords.robot
 Variables           ../../load_variables_from_profiles.py    ${CLUSTER_PROFILE}
-Library             odahuflow.robot.libraries.sdk_wrapper
 Library             odahuflow.robot.libraries.sdk_wrapper.ModelPackaging
 Library             odahuflow.robot.libraries.sdk_wrapper.ModelDeployment
 Library             odahuflow.robot.libraries.sdk_wrapper.ModelRoute
 Library             odahuflow.robot.libraries.sdk_wrapper.Model
 Suite Setup         Run Keywords
 ...                 Set Environment Variable  ODAHUFLOW_CONFIG  ${LOCAL_CONFIG}  AND
 ...                 Login to the api and edge  AND
-...                 Cleanup Resources
+...                 Cleanup All Resources
 Suite Teardown      Run Keywords
-...                 Cleanup Resources  AND
+...                 Cleanup All Resources  AND
 ...                 Remove File  ${LOCAL_CONFIG}
 Force Tags          api  sdk
```

Comment:
add `deployment` tag and set corresponding tags on other API_* tests plz

Reply:
I set tags (`deployment`/`route`/`model`) separately for every test. Or should I just set one tag for the whole test suite (`deployment`)?